presto-product-tests/conf/tempto/tempto-configuration.yaml

```diff
@@ -3,3 +3,23 @@ databases:
     host: hadoop-master
   presto:
     host: presto-master
+
+  prestoConfigUserAlice:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: alice
+    jdbc_password: na
+    jdbc_pooling: false
+
+  prestoConfigUserBob:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: bob
+    jdbc_password: na
+    jdbc_pooling: false
```

Comment:
Tempto runner is used from run_on_docker.

This conf file is used as a default when no conf file is given. Since
run_on_docker is using a different file, this one seems to be untested and
unused. I would create a separate PR to remove it. Add @arosa as reviewer
of it.

pon., 2.05.2016, 23:16 użytkownik Anu Sudarsan <notifications@github.com>
napisał:

> In presto-product-tests/conf/tempto/tempto-configuration.yaml
> <https://github.com/Teradata/presto/pull/204#discussion_r61806110>:
>
> > +    cli_server_address: ${databases.presto.host}:8080
> > +    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
> > +    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
> > +    jdbc_user: alice
> > +    jdbc_password: na
> > +    jdbc_pooling: false
> > +
> > +  prestoConfigUserBob:
> > +    host: localhost
> > +    server_address: ${databases.presto.host}:8080
> > +    cli_server_address: ${databases.presto.host}:8080
> > +    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
> > +    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
> > +    jdbc_user: bob
> > +    jdbc_password: na
> > +    jdbc_pooling: false
>
> conf/tempto/tempto-configuration.yaml is used by run_on_docker, and the
> other one I think was used when we used to run product tests using tempto
> runner <https://github.com/prestodb/tempto#using-tempto-runner>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Teradata/presto/pull/204/files/a6d086501a25dfd320ef974532c9669ef85b4bfe#r61806110>
>